runtime/src/prioritization_fee_cache.rs

```diff
@@ -0,0 +1,812 @@
+use {
+    crate::prioritization_fee::*,
+    crossbeam_channel::{unbounded, Receiver, Sender},
+    log::*,
+    solana_measure::measure,
+    solana_sdk::{
+        clock::Slot, pubkey::Pubkey, saturating_add_assign, transaction::SanitizedTransaction,
+    },
+    std::{
+        collections::HashMap,
+        sync::{
+            atomic::{AtomicU64, Ordering},
+            Arc, Mutex, RwLock,
+        },
+        thread::{Builder, JoinHandle},
+    },
+};
+
+/// The maximum number of blocks to keep in `PrioritizationFeeCache`; States from
+/// up to 150 recent blocks should be sufficient to estimate minimal prioritization fee to
+/// land transactions to current block.
+const MAX_NUM_RECENT_BLOCKS: u64 = 150;
+
+#[derive(Debug, Default)]
+struct PrioritizationFeeCacheMetrics {
+    // Count of transactions that successfully updated each slot's prioritization fee cache.
+    successful_transaction_update_count: AtomicU64,
```

Comment:
for the same reason, want to see if replayed transactions were successfully used to update fee cache, or failed due to reasons below. They are very unlikely to fail under normal condition; but if opinion is not strong, I still would like to keep these three counters for unexpected cases in real world. Otherwise, I am happy to remove them.